debian/bullseye/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:bullseye
+MAINTAINER Andrey Kulikov <amdeich@gmail.com>
+
+# Fix missing locales
+ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
+
+# Skip interactive post-install scripts
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Don't install recommends
+RUN echo 'apt::install-recommends "false";' > /etc/apt/apt.conf.d/00recommends
+
+RUN apt-get update && apt-get install -y --allow-downgrades \
```

Comment:
I understand that the `--allow-dowgrades` option is added instead of `--force-yes`. But I didn't find why `--force-yes` was added in the first place. I guess it was attempt to make apt-get fully non-interactive. However I didn't see any problem without the option, so I would remove it and wait for any real problem to come back.

Reply:
Removed. Please, let me know, if it'll lead to any problem for you.